plugins/modules/win_domain_computer.py

```diff
@@ -86,6 +89,28 @@
     type: str
     choices: [ absent, present ]
     default: present
+  offline_domain_join:
+    description:
+      - Provisions a computer in the directory and provides a BLOB file that can be used on the target computer/image to join it to the domain while offline.
+      - If the computer already exists, no BLOB will be created/returned, and the module will operate as it would have without offline domain join.
+    type: bool
+  odj_return_blob:
+    description:
+      - Returns the BLOB in the module output. Note that the BLOB should be treated as a secret, as it contains the machine password.
+      - The module should be called with C(no_log) when using this option.
+      - The BLOB file contains a terminating null character which will not be included. See Notes.
+  odj_blob_path:
+    description:
+      - The path to the file where the BLOB will be saved.
+      - Required when I(offline_domain_join=True) and I(odj_return_blob=False).
+      - If using I(odj_return_blob=True) this can be omitted, and a temporary file will be used and then destroyed.
+      - If I(odj_return_blob=True) and this is specified, it will be used and file will remain.
+notes:
+  - "For more information on Offline Domain Join
+    see U(https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/dd392267(v=ws.10))"
```

Comment:
You will probably have to remove the `(v=ws.10)` part of the URL. There's no way I know off to escape `(` and `)` in a URL for `U()`. Even better you can use `L(Link text,URL)` to shorten the URL as it is shown in the module docs.

Reply:
Will that cause a problem? I can't remove it completely (already tried, 404). I could try URL escape sequences, or I could use GitHub's URL shortener if that's acceptable.

See  my other comment about `L()`, could use some clarification there.

Reply:
> I can't remove it completely (already tried, 404)

That's weird, definitely works for other Microsoft docs but obviously not this one. You might have to use a URL shortened if you can't URL escape the brackets

Reply:
URL encodng the parentheses seemed to work (also switched to `L()`).